src/List.js

```diff
@@ -1,34 +1,46 @@
+import React from 'react'
+import {getAllMovies} from './apiCalls'
 import './List.css'
 
-const List = (props) => {
+class List extends React.Component {
```

Comment:
Class component looks good! This makes sense for List to have it's own state if it will be manipulating and using it directly.